src/test/java/teammates/test/cases/action/InstructorFeedbackResponseCommentDeleteActionTest.java

```diff
@@ -28,6 +28,7 @@ protected String getActionUri() {
     @Override
     @Test
     public void testExecuteAndPostProcess() {
+        prepareTestData();
```

Comment:
Here, we need to refresh the databundle so we can use `removeAndRestoreDataBundle(dataBundle);` 
in place of `prepareTestData();`